core/src/mast/serialization/decorator.rs

```diff
@@ -3,6 +3,8 @@ use alloc::vec::Vec;
 use miden_debug_types::{Location, Uri};
 use num_derive::{FromPrimitive, ToPrimitive};
 use num_traits::{FromPrimitive, ToPrimitive};
+#[cfg(all(feature = "arbitrary", feature = "std"))]
```

Comment:
Do we actually require gating on libstd for this? AFAIK we only require liballoc at most, but maybe I'm forgetting somewhere that needs libstd functionality in the `Arbitrary` impl.

Reply:
This is required in `proptest_derive` for instances of `Arbitrary`. Which should not be an actual issue since we actually use those `Arbitrary` instances in a context that has std imported through testing.